spec/discourse_automation_helper.rb

```diff
@@ -2,8 +2,31 @@
 
 require 'rails_helper'
 
+def capture_contexts(&blk)
+  DiscourseAutomation::CapturedContext.capture(&blk)
+end
+
+module DiscourseAutomation::CapturedContext
+  def self.add(context)
+    @contexts ||= []
```

Comment:
Is this check necessary? It feels like @contexts are always set by `self.capture`

Reply:
Yeah, this was an old remnant of my initial implementation, will clean up.